Automattic/wp-calypso
# Backup: Implement backup staging sites list API

## Proposed Changes
* Add necessary handlers and redux logic to implement new `rewind/staging/list` API to fetch the list of backup staging sites. Context: D107983-code

## To-do
- [x] Action types (API request life cycle)
- [x] API fetch handler
- [x] Reducers
- [x] Selectors 
- [x] Data query component -> `QueryBackupStagingSitesList` 

## Next steps
* Integrate the new `QueryBackupStagingSitesList` component to the [backup clone flow](https://github.com/Automattic/wp-calypso/blob/374b0dab88067d0aff22a9935365e7ea4057c586/client/my-sites/backup/clone-flow/index.tsx) that we started in #75571.
* Implement handlers and redux logic for marking a site as staging.
 
## Testing Instructions
This is an isolated change that will be integrated on [backup clone flow](https://github.com/Automattic/wp-calypso/blob/374b0dab88067d0aff22a9935365e7ea4057c586/client/my-sites/backup/clone-flow/index.tsx). So at this point you can ensure the unit tests are passing by using ```yarn run test-client client/state/rewind```.

## Pre-merge Checklist
